graylog2-web-interface/src/pages/LoginPage.jsx

```diff
@@ -68,18 +99,16 @@ const LoginPage = () => {
   };
 
   const renderLoginForm = () => {
-    const loginComponent = PluginStore.exports('loginProviderType');
-
-    if (loginComponent.length === 1) {
-      return React.createElement(loginComponent[0].formComponent, {
+    if (!useFallback && hasCustomLogin) {
+      return React.createElement(loginComponent.formComponent, {
```

Comment:
When an error occurs loading the component there's no way for the user to login back into Graylog. I'll write an issue for this.